ydb/core/tx/schemeshard/schemeshard_impl.cpp

```diff
@@ -1464,6 +1464,12 @@ const TTableInfo* TSchemeShard::GetMainTableForIndex(TPathId indexTableId) const
     if (!parentEl || !(*parentEl)->IsTableIndex())
         return nullptr;
 
+    if (index) {
+        if (const auto* indexInfo = Indexes.FindPtr(parentId)) {
```

Comment:
Why is it possible that we don't find `indexInfo`?
Maybe it should be an assert?

Reply:
Changed the code, now this comment is irrelevant.

Now we don't do the double check:
- path element IsTableIndex()
- Indexes contain the path ID
to define that the path ID represents an index.